extruct/w3cmicrodata.py

```diff
@@ -74,7 +74,7 @@ def extract(self, htmlstring, base_url=None, encoding="UTF-8"):
 
     def extract_items(self, document, base_url):
         itemids = self._build_itemids(document)
-        items_seen: Set[Any] = set()
+        items_seen: set[Any] = set()
```

Comment:
I'm curious how this works on Python 3.8 which we still support, I think this is a PEP 585 feature which was added in 3.9? https://peps.python.org/pep-0585/

Reply:
Ah never mind I can see `from __future__ import annotations` above